src/test/java/com/commercetools/sync/sdk2/products/utils/productupdateactionutils/BuildSetTaxCategoryUpdateActionTest.java

```diff
@@ -0,0 +1,76 @@
+package com.commercetools.sync.sdk2.products.utils.productupdateactionutils;
+
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildSetTaxCategoryUpdateAction;
+import static io.vrap.rmf.base.client.utils.json.JsonUtils.fromJsonString;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import com.commercetools.api.models.product.ProductDraft;
+import com.commercetools.api.models.product.ProductProjection;
+import com.commercetools.api.models.product.ProductSetTaxCategoryActionBuilder;
+import com.commercetools.api.models.tax_category.TaxCategoryReference;
+import com.commercetools.api.models.tax_category.TaxCategoryResourceIdentifier;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
Why do you need this annotation here? Is it because of `@Mock` annotation usage? Can you use `Mockito.mock()` like in other tests instead?

Reply:
[7782514](https://github.com/commercetools/commercetools-sync-java/pull/925/commits/778251427659ff026acf00c858e17b5bfc97e7ac)